meetings/2022-11-21.md

```diff
@@ -0,0 +1,110 @@
+# W3C Solid Community Group: Data Interoperability Panel
+
+* Date: 2022-11-21T15:00:00Z
+* Call: https://meet.jit.si/solid-data-interoperability
+* Chat: https://gitter.im/solid/data-interoperability-panel
+* Repository: https://github.com/solid/data-interoperability-panel
+
+## Present
+
+* elf Pavlik
+* Wouter Termont
+* Laurens Debackere
+* Justin Bingham
+
+### Regrets
+
+* ...
+
+---
+
+## Announcements
+
+### New meetings schedule
+
+eP: bi-weekly on Mondays at 15:00UTC => next meeting on 2022-11-21T15:00:00Z
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribe Selection
+
+* Pavlik
+
+## Agenda
+
+* Review open PRs
+* Discuss roadmap
+  * Authorization flow
+  * Get WAC on board
+  * Separate into smaller specs (out of time)
+  * Cleanup smaller issues (out of time)
+
+## Minutes
+
+### Review open PRs
+
+Overview: https://github.com/solid/data-interoperability-panel/pulls
+
+#### [#253 Add authz flow diagram for social agent sharing access](https://github.com/solid/data-interoperability-panel/pull/253)
+
+* eP: I'm concerned about creating access need in the app. Can we just pass IRI of the resource to AA?
+* WT: Would AA generate the access need?
+* eP: I think we don't need an access need here, it would be needed if Bob wants to request access from Alice. Here Alice initiates the flow.
```

Comment:
```suggestion
* eP: I think we don't need an access need here; it would be needed if Bob wants to request access from Alice. Here Alice initiates the flow.
```